src/common/util/intOps.ml

```diff
@@ -241,7 +241,7 @@ struct
 
   let shift_left = Z.shift_left
   let shift_right = Z.shift_right
-  let bitnot x = sub (neg x) one
+  let bitnot = Z.lognot
   let bitand = Z.logand
   let bitor = Z.logor
   let bitxor = Z.logxor
```

Comment:
Is this `bitnot` definition equivalent to the old one? Maybe there's some catch why it wasn't so before.

Also, we need to figure out what to do about this `bit*` vs `log*` confusion, where our naming differs from the standard library one (that Zarith also uses). I think a bunch of places now directly use `Z.logand` instead of `BI.bitand`, which may cause some confusion again.

One possibility would be to use `logand` for the bitwise operations to be consistent with OCaml. And then rename `bitand` to `c_logand` to emphasize the C nature of it.

Reply:
I was wondering the same, but then I checked the Zarith documentation, and there stands:
```
val lognot: t -> t
(** Bitwise logical negation. 
    The identity [lognot a]=[-a-1] always hold.
 *)
```
which at least by the definition seems to be the exact same operation.

Reply:
Renaming is fine by me 👍